third_party/terraform/utils/provider.go.erb

```diff
@@ -98,12 +97,45 @@ func Provider() terraform.ResourceProvider {
 			"google_project_services":                dataSourceGoogleProjectServices(),
 			"google_service_account":                 dataSourceGoogleServiceAccount(),
 			"google_service_account_key":             dataSourceGoogleServiceAccountKey(),
-			"google_storage_bucket_object":           dataSourceGoogleStorageBucketObject(),
```

Comment:
I think I shouldn't have removed these 2 data sources. They got pulled in as repeats because there is a data source and a resource.

Reply:
oh oops I totally assumed since my tests failed before cherrypicking and passed after it was all good haha.